### ISSUE TITLE
Suggestions with type number are converted to undefined

### ISSUE THREAD

**Issue by Hurtak** [bevacqua/horsey#6]
```js
horsey(document.getElementById('test'), {
	suggestions: ['aaa', 100, 200, 300]
});
```

will produce

http://i.imgur.com/QlqbUmr.png

**Comment by bevacqua** [comment]
These two methods should be fixed: https://github.com/bevacqua/horsey/blob/master/horsey.js#L502-L508

Care to create a PR?

**Comment by Hurtak** [comment]
Can't promise anything, but i will take a look at it tomorrow.

-----Original Message-----

**Comment by bevacqua** [comment]
Resolved in `2.5.2`
